helper/src/components/deployTab.js

```diff
@@ -27,30 +27,31 @@ export default function DeployTab({ defaults, updateFn, tabValues, invalidArray,
     ...(cluster.vmSize !== defaults.cluster.vmSize && { agentVMSize: cluster.vmSize }),
     ...(cluster.autoscale && { agentCountMax: cluster.maxCount }),
     ...(cluster.osDiskType === "Managed" && { osDiskType: cluster.osDiskType, ...(cluster.osDiskSizeGB > 0 && { osDiskSizeGB: cluster.osDiskSizeGB }) }),
-    ...(net.vnet_opt === "custom" && { 
-         custom_vnet: true, 
-         ...serviceparams, 
+    ...(net.vnet_opt === "custom" && {
+         custom_vnet: true,
+         ...serviceparams,
          ...aksvnetparams,
          ...(net.bastion !== defaults.net.bastion && {bastion: net.bastion}),
          ...(net.bastion && defaults.net.bastionSubnetAddressPrefix !== net.bastionSubnetAddressPrefix && {bastionSubnetAddressPrefix: net.bastionSubnetAddressPrefix})
        }),
     ...(net.vnet_opt === "byo" && { byoAKSSubnetId: net.byoAKSSubnetId, ...serviceparams }),
     ...(net.vnet_opt === "byo" && addons.ingress === 'appgw' && { byoAGWSubnetId: net.byoAGWSubnetId }),
     ...(cluster.enable_aad && { enable_aad: true, ...(cluster.enableAzureRBAC === false && cluster.aad_tenant_id && { aad_tenant_id: cluster.aad_tenant_id }) }),
+    ...(cluster.enable_aad && cluster.AksDisableLocalAccounts && { AksDisableLocalAccounts: cluster.AksDisableLocalAccounts }),
```

Comment:
This is fine if the default is false, but will break if we switch the default to true,  maybe : ...(cluster.enable_aad && cluster.AksDisableLocalAccounts !== defaults.cluster.AksDisableLocalAccount && { AksDisableLocalAccounts: cluster.AksDisableLocalAccounts })

Reply:
Makes sense. Thanks for the catch 👍

Reply:
Although @khowling - i totally just copied your extra statement and didn't notice the variable name was correct.
Thank goodness for Playwright.  It caught the behaviour of the checkbox -> az cmd wasn't right. 

🚀🚀🚀🚀🚀🚀